### DISCUSSION TITLE
Template projects

### DISCUSSION THREAD

**Discussion by KingCreeperz** [ScratchAddons/ScratchAddons#4890] [Ideas]
This is kinda already a feature with remixing but a addon to choose a template every time you make a project. So it automatically copies/remixes the project for the creator to start working.

**Comment by BlueDome77** [comment]
There is already an addon for this: Customizable default project

**Reply by KingCreeperz** [reply]
Yes I know this exists, but imagine a screen that is built into scratch addons that shows multiple templates when you make a project, works kinda like roblox’s template system that shows multiple templates on the create menu on the main menu. Hope that cleared out what I was trying to say!

**Comment by Fluffy728** [comment]
Support. I kinda may not need it but maybe other people

**Reply by Samq64** [reply]
Since Github has reactions comments that just say "suppprt" aren't necessary.
